Further reduce use of memset()

cdumez opened the pull request:
#### 98af39ad6c25944145ada5d30f5c1f1db2e9d35c
<pre>
Further reduce use of memset()
<a href="https://bugs.webkit.org/show_bug.cgi?id=284605">https://bugs.webkit.org/show_bug.cgi?id=284605</a>

Reviewed by Darin Adler.

* Source/WebCore/platform/mediastream/mac/CoreAudioCaptureDeviceManager.cpp:
(WebCore::deviceHasStreams):
* Source/WebKitLegacy/mac/WebView/WebDynamicScrollBarsView.mm:
(-[WebDynamicScrollBarsView initWithFrame:]):
(-[WebDynamicScrollBarsView initWithCoder:]):
* Tools/TestRunnerShared/IOSLayoutTestCommunication.cpp:
(setUpIOSLayoutTestCommunication):
* Tools/TestWebKitAPI/Tests/InjectInternals_Bundle.cpp:
(TestWebKitAPI::InternalsInjectedBundleTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WTF/cocoa/URLExtras.mm:
(TestWebKitAPI::TEST(WTF_URLExtras, URLExtras_ParsingError)):
* Tools/TestWebKitAPI/Tests/WebCore/CARingBufferTest.cpp:
(TestWebKitAPI::TEST_F(CARingBufferTest, Basics)):
(TestWebKitAPI::MixingTest::run):
* Tools/TestWebKitAPI/Tests/WebCore/LineBreaking.mm:
(breakingLocationsFromICU):
* Tools/TestWebKitAPI/Tests/WebKit/AboutBlankLoad.cpp:
(TestWebKitAPI::TEST(WebKit, AboutBlankLoad)):
* Tools/TestWebKitAPI/Tests/WebKit/CanHandleRequest.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/CloseFromWithinCreatePage.cpp:
(TestWebKitAPI::createNewPageThenClose):
(TestWebKitAPI::TEST(WebKit, CloseFromWithinCreatePage)):
(TestWebKitAPI::createNewPage):
(TestWebKitAPI::TEST(WebKit, CreatePageThenDocumentOpenMIMEType)):
* Tools/TestWebKitAPI/Tests/WebKit/CloseThenTerminate.cpp:
(TestWebKitAPI::TEST(WebKit, CloseThenTerminate)):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionBasic.cpp:
(TestWebKitAPI::TEST(WebKit, DISABLED_DOMWindowExtensionBasic)):
(TestWebKitAPI::TEST(WebKit, DOMWindowExtensionCrashOnReload)):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionBasic_Bundle.cpp:
(TestWebKitAPI::DOMWindowExtensionBasic::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionNoCache.cpp:
(TestWebKitAPI::TEST(WebKit, DISABLED_DOMWindowExtensionNoCache)):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionNoCache_Bundle.cpp:
(TestWebKitAPI::DOMWindowExtensionNoCache::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DeferredViewInWindowStateChange.mm:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/DidAssociateFormControls.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/DidAssociateFormControls_Bundle.cpp:
(TestWebKitAPI::DidAssociateFormControlsTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DidNotHandleKeyDown.cpp:
(TestWebKitAPI::TEST(WebKit, DidNotHandleKeyDown)):
* Tools/TestWebKitAPI/Tests/WebKit/DidRemoveFrameFromHiearchyInPageCache.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/DidRemoveFrameFromHiearchyInPageCache_Bundle.cpp:
(TestWebKitAPI::DidRemoveFrameFromHiearchyInBackForwardCacheTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DocumentStartUserScriptAlertCrash.cpp:
(TestWebKitAPI::TEST(WebKit, DocumentStartUserScriptAlertCrashTest)):
* Tools/TestWebKitAPI/Tests/WebKit/DownloadDecideDestinationCrash.cpp:
(TestWebKitAPI::setPagePolicyClient):
* Tools/TestWebKitAPI/Tests/WebKit/EnumerateMediaDevices.cpp:
(TestWebKitAPI::TEST(WebKit, EnumerateDevices)):
* Tools/TestWebKitAPI/Tests/WebKit/EphemeralSessionPushStateNoHistoryCallback.cpp:
(TestWebKitAPI::TEST(WebKit, EphemeralSessionPushStateNoHistoryCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/EventModifiers.cpp:
(TestWebKitAPI::setClients):
* Tools/TestWebKitAPI/Tests/WebKit/FailedLoad.cpp:
(TestWebKitAPI::TEST(WebKit, FailedLoad)):
* Tools/TestWebKitAPI/Tests/WebKit/Find.cpp:
(TestWebKitAPI::TEST(WebKit, Find)):
* Tools/TestWebKitAPI/Tests/WebKit/FindMatches.mm:
(TestWebKitAPI::TEST(WebKit, FindMatches)):
* Tools/TestWebKitAPI/Tests/WebKit/FirstMeaningfulPaintMilestone.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/ForceRepaint.cpp:
(TestWebKitAPI::TEST(WebKit, ForceRepaint)):
* Tools/TestWebKitAPI/Tests/WebKit/FrameMIMETypeHTML.cpp:
(TestWebKitAPI::TEST(WebKit, FrameMIMETypeHTML)):
* Tools/TestWebKitAPI/Tests/WebKit/FrameMIMETypePNG.cpp:
(TestWebKitAPI::TEST(WebKit, FrameMIMETypePNG)):
* Tools/TestWebKitAPI/Tests/WebKit/Geolocation.cpp:
(TestWebKitAPI::setupGeolocationProvider):
(TestWebKitAPI::setupView):
(TestWebKitAPI::TEST(WebKit, GeolocationTransitionToLowAccuracy)):
(TestWebKitAPI::TEST(WebKit, GeolocationWatchMultiprocess)):
* Tools/TestWebKitAPI/Tests/WebKit/GetInjectedBundleInitializationUserDataCallback.cpp:
(TestWebKitAPI::TEST(WebKit, GetInjectedBundleInitializationUserDataCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/HitTestResultNodeHandle.cpp:
(TestWebKitAPI::setPageLoaderClient):
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/HitTestResultNodeHandle_Bundle.cpp:
(TestWebKitAPI::HitTestResultNodeHandleTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleBasic.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleBasic)):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleDisableOverrideBuiltinsBehavior.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleNoDisableOverrideBuiltinsBehaviorTest)):
(TestWebKitAPI::TEST(WebKit, InjectedBundleDisableOverrideBuiltinsBehaviorTest)):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleFrameHitTest.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleFrameHitTest_Bundle.cpp:
(TestWebKitAPI::InjectedBundleFrameHitTestTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleInitializationUserDataCallbackWins.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleInitializationUserDataCallbackWins)):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleMakeAllShadowRootsOpen.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleMakeAllShadowRootOpenTest)):
* Tools/TestWebKitAPI/Tests/WebKit/LayoutMilestonesWithAllContentInFrame.cpp:
(TestWebKitAPI::TEST(WebKit, LayoutMilestonesWithAllContentInFrame)):
(TestWebKitAPI::TEST(WebKit, FirstVisuallyNonEmptyLayoutAfterPageCacheRestore)):
(TestWebKitAPI::TEST(WebKit, FirstVisuallyNonEmptyMilestoneWithLoadComplete)):
* Tools/TestWebKitAPI/Tests/WebKit/LoadAlternateHTMLStringWithNonDirectoryURL.cpp:
(TestWebKitAPI::loadAlternateHTMLString):
* Tools/TestWebKitAPI/Tests/WebKit/LoadCanceledNoServerRedirectCallback.cpp:
(TestWebKitAPI::TEST(WebKit, LoadCanceledNoServerRedirectCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/LoadCanceledNoServerRedirectCallback_Bundle.cpp:
(TestWebKitAPI::LoadCanceledNoServerRedirectCallbackTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/LoadPageOnCrash.cpp:
(TestWebKitAPI::WebKit2CrashLoader::WebKit2CrashLoader):
* Tools/TestWebKitAPI/Tests/WebKit/MenuTypesForMouseEvents.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/ModalAlertsSPI.cpp:
(TestWebKitAPI::createNewPage):
(TestWebKitAPI::TEST(WebKit, ModalAlertsSPI)):
(TestWebKitAPI::TEST(WebKit, CreateNewPageDelegateFrameLoadState)):
* Tools/TestWebKitAPI/Tests/WebKit/MouseMoveAfterCrash.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NavigationClientDefaultCrypto.cpp:
(TestWebKitAPI::TEST(WebKit, NavigationClientDefaultCrypto)):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayout.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayoutFails.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayoutForImages.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayoutFrames.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/PageLoadBasic.cpp:
(TestWebKitAPI::TEST(WebKit, PageLoadBasic)):
(TestWebKitAPI::TEST(WebKit, PageLoadTwiceAndReload)):
* Tools/TestWebKitAPI/Tests/WebKit/PageLoadDidChangeLocationWithinPageForFrame.cpp:
(TestWebKitAPI::TEST(WebKit, PageLoadDidChangeLocationWithinPage)):
* Tools/TestWebKitAPI/Tests/WebKit/ParentFrame.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/ParentFrame_Bundle.cpp:
(TestWebKitAPI::ParentFrameTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/PasteboardNotifications.mm:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/PasteboardNotifications_Bundle.cpp:
(TestWebKitAPI::PasteboardNotificationsTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/PendingAPIRequestURL.cpp:
(TestWebKitAPI::TEST(WebKit, PendingAPIRequestURL)):
* Tools/TestWebKitAPI/Tests/WebKit/PrivateBrowsingPushStateNoHistoryCallback.cpp:
(TestWebKitAPI::TEST(WebKit, PrivateBrowsingPushStateNoHistoryCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/ProcessDidTerminate.cpp:
(TestWebKitAPI::TEST(WebKit, ProcessDidTerminateRequestedByClient)):
(TestWebKitAPI::TEST(WebKit, ProcessDidTerminateWithReasonCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/ProvisionalURLAfterWillSendRequestCallback.cpp:
(TestWebKitAPI::TEST(WebKit2, ProvisionalURLAfterWillSendRequestCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/ProvisionalURLAfterWillSendRequestCallback_Bundle.cpp:
* Tools/TestWebKitAPI/Tests/WebKit/ReloadPageAfterCrash.cpp:
(TestWebKitAPI::TEST(WebKit, ReloadPageAfterCrash)):
(TestWebKitAPI::TEST(WebKit, FocusedFrameAfterCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/ResizeReversePaginatedWebView.cpp:
(TestWebKitAPI::TEST(WebKit, ResizeReversePaginatedWebView)):
* Tools/TestWebKitAPI/Tests/WebKit/ResizeWindowAfterCrash.cpp:
(TestWebKitAPI::TEST(WebKit, ResizeWindowAfterCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/RestoreSessionState.cpp:
(TestWebKitAPI::setPageLoaderClient):
(TestWebKitAPI::TEST(WebKit, RestoreSessionStateContainingScrollRestorationDefaultWithAsyncPolicyDelegates)):
(TestWebKitAPI::TEST(WebKit, ClearedPendingURLAfterCancelingRestoreSessionState)):
* Tools/TestWebKitAPI/Tests/WebKit/RestoreSessionStateContainingFormData.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/ScrollPinningBehaviors.mm:
(TestWebKitAPI::TEST(WebKit, ScrollPinningBehaviors)):
* Tools/TestWebKitAPI/Tests/WebKit/ShouldKeepCurrentBackForwardListItemInList.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/SpacebarScrolling.cpp:
(TestWebKitAPI::TEST(WebKit, SpacebarScrolling)):
* Tools/TestWebKitAPI/Tests/WebKit/TerminateTwice.cpp:
(TestWebKitAPI::TEST(WebKit, TerminateTwice)):
* Tools/TestWebKitAPI/Tests/WebKit/TextFieldDidBeginAndEndEditing.cpp:
(TestWebKitAPI::WebKit2TextFieldBeginAndEditEditingTest::setInjectedBundleClient):
(TestWebKitAPI::WebKit2TextFieldBeginAndEditEditingTest::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/TextFieldDidBeginAndEndEditing_Bundle.cpp:
(TestWebKitAPI::TextFieldDidBeginAndEndEditingEventsTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/UserMedia.cpp:
(TestWebKitAPI::TEST(WebKit, UserMediaBasic)):
(TestWebKitAPI::TEST(WebKit, OnDeviceChangeCrash)):
(TestWebKitAPI::TEST(WebKit, EnumerateDevicesCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/UserMessage.cpp:
(TestWebKitAPI::WebKit2UserMessageRoundTripTest::setInjectedBundleClient):
(TestWebKitAPI::WebKit2UserMessageRoundTripTest::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageConfiguration.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageCopySessionStateWithFiltering.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageGetScaleFactorNotZero.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageIsPlayingAudio.cpp:
(TestWebKitAPI::setUpClients):
* Tools/TestWebKitAPI/Tests/WebKit/WKThumbnailView.mm:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WebArchive.cpp:
(TestWebKitAPI::setInjectedBundleClient):
(TestWebKitAPI::TEST(WebKit, WebArchive)):
* Tools/TestWebKitAPI/Tests/WebKit/WillLoad.cpp:
(TestWebKitAPI::WebKit2WillLoadTest::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/WillLoad_Bundle.cpp:
* Tools/TestWebKitAPI/Tests/WebKit/WillSendSubmitEvent.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/WillSendSubmitEvent_Bundle.cpp:
(TestWebKitAPI::WillSendSubmitEventTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/mac/AttributedSubstringForProposedRangeWithImage.mm:
(TestWebKitAPI::TEST(WebKit, AttributedSubstringForProposedRangeWithImage)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/ContextMenuDownload.mm:
(TestWebKitAPI::TEST(WebKit, ContextMenuDownloadHTMLDownloadAttribute)):
(TestWebKitAPI::TEST(WebKit, ContextMenuDownloadHTMLDownloadAttributeWithSlashes)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/CustomBundleParameter.mm:
(TestWebKitAPI::TEST(WebKit, CustomBundleParameter)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/EditorCommands.mm:
(TestWebKitAPI::TEST(WebKit, ScrollByLineCommands)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/ForceLightAppearanceInBundle.mm:
(TestWebKitAPI::TEST(WebKit, ForceLightAppearanceInBundle)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/GetBackingScaleFactor.mm:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/mac/GetPIDAfterAbortedProcessLaunch.cpp:
(TestWebKitAPI::TEST(WebKit, GetPIDAfterAbortedProcessLaunch)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/InjectedBundleAppleEvent.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleAppleEvent)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/RestoreStateAfterTermination.mm:
(TestWebKitAPI::TEST(WebKit, RestoreStateAfterTermination)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/CommandBackForward.mm:
(WebKit2_CommandBackForwardTestWKView::SetUp):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/DecidePolicyForNavigationAction.mm:
(TEST(WebKit, DecidePolicyForNewWindowAction)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/PictureInPictureDelegate.mm:
(TestWebKitAPI::TEST(PictureInPicture, WKPageUIClient)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/ResponsivenessTimerDoesntFireEarly.mm:
(setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/UIDelegate.mm:
(TEST(WebKit, GeolocationPermission)):
((WebKit, GeolocationPermissionInIFrame)):
((WebKit, GeolocationPermissionInDisallowedIFrame)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/WKURLSchemeHandler-1.mm:
((URLSchemeHandler, XHRPost)):
* Tools/TestWebKitAPI/Tests/WebKitGLib/TestUIClient.cpp:
* Tools/TestWebKitAPI/Tests/WebKitObjC/PreventImageLoadWithAutoResizing_Bundle.cpp:
(TestWebKitAPI::DenyWillSendRequestTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/mac/FirstResponderScrollingPosition.mm:
(TestWebKitAPI::TEST(WebKit, DISABLED_FirstResponderScrollingPosition)):
* Tools/TestWebKitAPI/Tests/mac/FullscreenZoomInitialFrame.mm:
(TestWebKitAPI::FullscreenZoomInitialFrame::initializeView):
* Tools/TestWebKitAPI/Tests/mac/PageVisibilityStateWithWindowChanges.mm:
(TestWebKitAPI::PageVisibilityStateWithWindowChanges::initializeView):
* Tools/TestWebKitAPI/mac/WebKitAgnosticTest.mm:
(TestWebKitAPI::setPageLoaderClient):
* Tools/WebKitTestRunner/GeolocationProviderMock.cpp:
(WTR::GeolocationProviderMock::GeolocationProviderMock):

Canonical link: <a href="https://commits.webkit.org/287824@main">https://commits.webkit.org/287824@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/5b14f8cd4dd85e988ccc376637963aaf9e6d0e83

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/80938 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/466 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/34883 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/85469 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/31926 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/83049 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/478 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/8261 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/63209 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/20980 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/84007 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/279 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/73679 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/43507 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/209 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/27836 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/30384 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/73927 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/71736 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/28385 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/86904 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/80006 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/8170 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/5778 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/71519 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/8347 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/69511 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/70753 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/14777 "Passed tests") | [❌ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/13707 "Found 1 new test failure: fast/mediastream/device-change-event-2.html (failure)") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/102404 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/12562 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/8134 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/13654 "Built successfully") | [![loading-orange](https://github-production-user-asset-6210df.s3.amazonaws.com/3098702/291015173-08c448be-ac0a-4fd6-92a3-8165057445b7.png) 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/24886 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Found 3 jsc stress test failures: stress/json-stringify-inspector-check.js.no-llint, wasm.yaml/wasm/stress/cc-int-to-int-cross-module-with-exception.js.default-wasm, wasm.yaml/wasm/stress/struct-new_default-small-members.js.wasm-eager-jettison; Compiled JSC; jscore-tests; Running analyze-jsc-tests-results") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/7968 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/11488 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/9776 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

darinadler commented on Tools/TestWebKitAPI/Tests/WTF/cocoa/URLExtras.mm:
```diff
@@ -312,11 +312,10 @@
     WTF::URL url5 { "file:///A%C3%A7%C3%A3o.html"_str };
-    CFURLGetFileSystemRepresentation(url5.createCFURL().get(), false, reinterpret_cast<UInt8*>(buffer), sizeof(buffer));
-    EXPECT_STREQ(buffer, "/Ação.html");
+    CFURLGetFileSystemRepresentation(url5.createCFURL().get(), false, reinterpret_cast<UInt8*>(buffer.data()), buffer.size());
```
Should use `byteCast`.

darinadler commented on Tools/TestWebKitAPI/Tests/WebCore/CARingBufferTest.cpp:
```diff
@@ -105,11 +105,11 @@ TEST_F(CARingBufferTest, Basics)
         sourceBuffer[i] = i + 0.5;
 
-    setListDataBuffer(reinterpret_cast<uint8_t*>(sourceBuffer), capacity);
+    setListDataBuffer(reinterpret_cast<uint8_t*>(sourceBuffer.data()), capacity);
```
We should move this `reinterpret_cast` and call to `data()` inside the `setListDataBuffer` function. But I guess there's a limit to how much I care about that in test code.

darinadler reviewed (approved):
I think it's a little inelegant to pass a `0` byte value into a function when we are always using it to zero things out. I know that's how `memset` works but we are using our own function so we aren't stuck with that design. We could have a `zeroSpan` function and maybe also a `zeroBytes` function template that does the sizing that `asMutableByteSpan` does. I suspect we don't need `memsetSpan` at all since I bet we never call it with anything other than `0`.

darinadler commented on Source/WebCore/Modules/compression/ZStream.cpp:
```diff
@@ -74,7 +74,7 @@ bool ZStream::initializeIfNecessary(Algorithm algorithm, Operation operation)
 ZStream::ZStream()
 {
-    memsetSpan(asMutableByteSpan(m_stream), 0);
+    zeroSpan(asMutableByteSpan(m_stream));
```
This can use `zeroBytes`.

darinadler commented on Source/WebCore/platform/audio/AudioArray.h:
```diff
@@ -102,7 +102,7 @@ class AudioArray {
     const T& operator[](size_t i) const { return at(i); }
 
-    void zero() { memsetSpan(this->span(), 0); }
+    void zero() { zeroSpan(this->span()); }
```
Why is `this->` needed?

cdumez replied:
No, I've dropped a few `this->` from this header.

darinadler commented on Source/WebCore/platform/audio/AudioArray.h:
```diff
@@ -113,7 +113,7 @@ class AudioArray {
         // This expression cannot overflow because end - start cannot be
         // greater than m_size, which is safe due to the check in resize().
-        memsetSpan(this->span().subspan(start, end - start), 0);
+        zeroSpan(this->span().subspan(start, end - start));
```
Why is `this->` needed?

darinadler commented on Tools/TestWebKitAPI/Tests/WebCore/CARingBufferTest.cpp:
```diff
@@ -62,19 +62,21 @@ class CARingBufferTest : public testing::Test {
-            if (bufferData)
-                bufferData = bufferData + bytesPerChannel;
+            m_bufferList->mBuffers[i].mData = bufferBytes.data();
+            if (bufferBytes.data())
```
I don’t think this can ever be null.

cdumez committed:
Further reduce use of memset()
https://bugs.webkit.org/show_bug.cgi?id=284605

Reviewed by Darin Adler.

* Source/WebCore/platform/mediastream/mac/CoreAudioCaptureDeviceManager.cpp:
(WebCore::deviceHasStreams):
* Source/WebKitLegacy/mac/WebView/WebDynamicScrollBarsView.mm:
(-[WebDynamicScrollBarsView initWithFrame:]):
(-[WebDynamicScrollBarsView initWithCoder:]):
* Tools/TestRunnerShared/IOSLayoutTestCommunication.cpp:
(setUpIOSLayoutTestCommunication):
* Tools/TestWebKitAPI/Tests/InjectInternals_Bundle.cpp:
(TestWebKitAPI::InternalsInjectedBundleTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WTF/cocoa/URLExtras.mm:
(TestWebKitAPI::TEST(WTF_URLExtras, URLExtras_ParsingError)):
* Tools/TestWebKitAPI/Tests/WebCore/CARingBufferTest.cpp:
(TestWebKitAPI::TEST_F(CARingBufferTest, Basics)):
(TestWebKitAPI::MixingTest::run):
* Tools/TestWebKitAPI/Tests/WebCore/LineBreaking.mm:
(breakingLocationsFromICU):
* Tools/TestWebKitAPI/Tests/WebKit/AboutBlankLoad.cpp:
(TestWebKitAPI::TEST(WebKit, AboutBlankLoad)):
* Tools/TestWebKitAPI/Tests/WebKit/CanHandleRequest.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/CloseFromWithinCreatePage.cpp:
(TestWebKitAPI::createNewPageThenClose):
(TestWebKitAPI::TEST(WebKit, CloseFromWithinCreatePage)):
(TestWebKitAPI::createNewPage):
(TestWebKitAPI::TEST(WebKit, CreatePageThenDocumentOpenMIMEType)):
* Tools/TestWebKitAPI/Tests/WebKit/CloseThenTerminate.cpp:
(TestWebKitAPI::TEST(WebKit, CloseThenTerminate)):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionBasic.cpp:
(TestWebKitAPI::TEST(WebKit, DISABLED_DOMWindowExtensionBasic)):
(TestWebKitAPI::TEST(WebKit, DOMWindowExtensionCrashOnReload)):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionBasic_Bundle.cpp:
(TestWebKitAPI::DOMWindowExtensionBasic::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionNoCache.cpp:
(TestWebKitAPI::TEST(WebKit, DISABLED_DOMWindowExtensionNoCache)):
* Tools/TestWebKitAPI/Tests/WebKit/DOMWindowExtensionNoCache_Bundle.cpp:
(TestWebKitAPI::DOMWindowExtensionNoCache::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DeferredViewInWindowStateChange.mm:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/DidAssociateFormControls.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/DidAssociateFormControls_Bundle.cpp:
(TestWebKitAPI::DidAssociateFormControlsTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DidNotHandleKeyDown.cpp:
(TestWebKitAPI::TEST(WebKit, DidNotHandleKeyDown)):
* Tools/TestWebKitAPI/Tests/WebKit/DidRemoveFrameFromHiearchyInPageCache.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/DidRemoveFrameFromHiearchyInPageCache_Bundle.cpp:
(TestWebKitAPI::DidRemoveFrameFromHiearchyInBackForwardCacheTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/DocumentStartUserScriptAlertCrash.cpp:
(TestWebKitAPI::TEST(WebKit, DocumentStartUserScriptAlertCrashTest)):
* Tools/TestWebKitAPI/Tests/WebKit/DownloadDecideDestinationCrash.cpp:
(TestWebKitAPI::setPagePolicyClient):
* Tools/TestWebKitAPI/Tests/WebKit/EnumerateMediaDevices.cpp:
(TestWebKitAPI::TEST(WebKit, EnumerateDevices)):
* Tools/TestWebKitAPI/Tests/WebKit/EphemeralSessionPushStateNoHistoryCallback.cpp:
(TestWebKitAPI::TEST(WebKit, EphemeralSessionPushStateNoHistoryCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/EventModifiers.cpp:
(TestWebKitAPI::setClients):
* Tools/TestWebKitAPI/Tests/WebKit/FailedLoad.cpp:
(TestWebKitAPI::TEST(WebKit, FailedLoad)):
* Tools/TestWebKitAPI/Tests/WebKit/Find.cpp:
(TestWebKitAPI::TEST(WebKit, Find)):
* Tools/TestWebKitAPI/Tests/WebKit/FindMatches.mm:
(TestWebKitAPI::TEST(WebKit, FindMatches)):
* Tools/TestWebKitAPI/Tests/WebKit/FirstMeaningfulPaintMilestone.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/ForceRepaint.cpp:
(TestWebKitAPI::TEST(WebKit, ForceRepaint)):
* Tools/TestWebKitAPI/Tests/WebKit/FrameMIMETypeHTML.cpp:
(TestWebKitAPI::TEST(WebKit, FrameMIMETypeHTML)):
* Tools/TestWebKitAPI/Tests/WebKit/FrameMIMETypePNG.cpp:
(TestWebKitAPI::TEST(WebKit, FrameMIMETypePNG)):
* Tools/TestWebKitAPI/Tests/WebKit/Geolocation.cpp:
(TestWebKitAPI::setupGeolocationProvider):
(TestWebKitAPI::setupView):
(TestWebKitAPI::TEST(WebKit, GeolocationTransitionToLowAccuracy)):
(TestWebKitAPI::TEST(WebKit, GeolocationWatchMultiprocess)):
* Tools/TestWebKitAPI/Tests/WebKit/GetInjectedBundleInitializationUserDataCallback.cpp:
(TestWebKitAPI::TEST(WebKit, GetInjectedBundleInitializationUserDataCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/HitTestResultNodeHandle.cpp:
(TestWebKitAPI::setPageLoaderClient):
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/HitTestResultNodeHandle_Bundle.cpp:
(TestWebKitAPI::HitTestResultNodeHandleTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleBasic.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleBasic)):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleDisableOverrideBuiltinsBehavior.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleNoDisableOverrideBuiltinsBehaviorTest)):
(TestWebKitAPI::TEST(WebKit, InjectedBundleDisableOverrideBuiltinsBehaviorTest)):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleFrameHitTest.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleFrameHitTest_Bundle.cpp:
(TestWebKitAPI::InjectedBundleFrameHitTestTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleInitializationUserDataCallbackWins.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleInitializationUserDataCallbackWins)):
* Tools/TestWebKitAPI/Tests/WebKit/InjectedBundleMakeAllShadowRootsOpen.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleMakeAllShadowRootOpenTest)):
* Tools/TestWebKitAPI/Tests/WebKit/LayoutMilestonesWithAllContentInFrame.cpp:
(TestWebKitAPI::TEST(WebKit, LayoutMilestonesWithAllContentInFrame)):
(TestWebKitAPI::TEST(WebKit, FirstVisuallyNonEmptyLayoutAfterPageCacheRestore)):
(TestWebKitAPI::TEST(WebKit, FirstVisuallyNonEmptyMilestoneWithLoadComplete)):
* Tools/TestWebKitAPI/Tests/WebKit/LoadAlternateHTMLStringWithNonDirectoryURL.cpp:
(TestWebKitAPI::loadAlternateHTMLString):
* Tools/TestWebKitAPI/Tests/WebKit/LoadCanceledNoServerRedirectCallback.cpp:
(TestWebKitAPI::TEST(WebKit, LoadCanceledNoServerRedirectCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/LoadCanceledNoServerRedirectCallback_Bundle.cpp:
(TestWebKitAPI::LoadCanceledNoServerRedirectCallbackTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/LoadPageOnCrash.cpp:
(TestWebKitAPI::WebKit2CrashLoader::WebKit2CrashLoader):
* Tools/TestWebKitAPI/Tests/WebKit/MenuTypesForMouseEvents.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/ModalAlertsSPI.cpp:
(TestWebKitAPI::createNewPage):
(TestWebKitAPI::TEST(WebKit, ModalAlertsSPI)):
(TestWebKitAPI::TEST(WebKit, CreateNewPageDelegateFrameLoadState)):
* Tools/TestWebKitAPI/Tests/WebKit/MouseMoveAfterCrash.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NavigationClientDefaultCrypto.cpp:
(TestWebKitAPI::TEST(WebKit, NavigationClientDefaultCrypto)):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayout.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayoutFails.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayoutForImages.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/NewFirstVisuallyNonEmptyLayoutFrames.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/PageLoadBasic.cpp:
(TestWebKitAPI::TEST(WebKit, PageLoadBasic)):
(TestWebKitAPI::TEST(WebKit, PageLoadTwiceAndReload)):
* Tools/TestWebKitAPI/Tests/WebKit/PageLoadDidChangeLocationWithinPageForFrame.cpp:
(TestWebKitAPI::TEST(WebKit, PageLoadDidChangeLocationWithinPage)):
* Tools/TestWebKitAPI/Tests/WebKit/ParentFrame.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/ParentFrame_Bundle.cpp:
(TestWebKitAPI::ParentFrameTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/PasteboardNotifications.mm:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/PasteboardNotifications_Bundle.cpp:
(TestWebKitAPI::PasteboardNotificationsTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/PendingAPIRequestURL.cpp:
(TestWebKitAPI::TEST(WebKit, PendingAPIRequestURL)):
* Tools/TestWebKitAPI/Tests/WebKit/PrivateBrowsingPushStateNoHistoryCallback.cpp:
(TestWebKitAPI::TEST(WebKit, PrivateBrowsingPushStateNoHistoryCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/ProcessDidTerminate.cpp:
(TestWebKitAPI::TEST(WebKit, ProcessDidTerminateRequestedByClient)):
(TestWebKitAPI::TEST(WebKit, ProcessDidTerminateWithReasonCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/ProvisionalURLAfterWillSendRequestCallback.cpp:
(TestWebKitAPI::TEST(WebKit2, ProvisionalURLAfterWillSendRequestCallback)):
* Tools/TestWebKitAPI/Tests/WebKit/ProvisionalURLAfterWillSendRequestCallback_Bundle.cpp:
* Tools/TestWebKitAPI/Tests/WebKit/ReloadPageAfterCrash.cpp:
(TestWebKitAPI::TEST(WebKit, ReloadPageAfterCrash)):
(TestWebKitAPI::TEST(WebKit, FocusedFrameAfterCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/ResizeReversePaginatedWebView.cpp:
(TestWebKitAPI::TEST(WebKit, ResizeReversePaginatedWebView)):
* Tools/TestWebKitAPI/Tests/WebKit/ResizeWindowAfterCrash.cpp:
(TestWebKitAPI::TEST(WebKit, ResizeWindowAfterCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/RestoreSessionState.cpp:
(TestWebKitAPI::setPageLoaderClient):
(TestWebKitAPI::TEST(WebKit, RestoreSessionStateContainingScrollRestorationDefaultWithAsyncPolicyDelegates)):
(TestWebKitAPI::TEST(WebKit, ClearedPendingURLAfterCancelingRestoreSessionState)):
* Tools/TestWebKitAPI/Tests/WebKit/RestoreSessionStateContainingFormData.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/ScrollPinningBehaviors.mm:
(TestWebKitAPI::TEST(WebKit, ScrollPinningBehaviors)):
* Tools/TestWebKitAPI/Tests/WebKit/ShouldKeepCurrentBackForwardListItemInList.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/SpacebarScrolling.cpp:
(TestWebKitAPI::TEST(WebKit, SpacebarScrolling)):
* Tools/TestWebKitAPI/Tests/WebKit/TerminateTwice.cpp:
(TestWebKitAPI::TEST(WebKit, TerminateTwice)):
* Tools/TestWebKitAPI/Tests/WebKit/TextFieldDidBeginAndEndEditing.cpp:
(TestWebKitAPI::WebKit2TextFieldBeginAndEditEditingTest::setInjectedBundleClient):
(TestWebKitAPI::WebKit2TextFieldBeginAndEditEditingTest::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/TextFieldDidBeginAndEndEditing_Bundle.cpp:
(TestWebKitAPI::TextFieldDidBeginAndEndEditingEventsTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/UserMedia.cpp:
(TestWebKitAPI::TEST(WebKit, UserMediaBasic)):
(TestWebKitAPI::TEST(WebKit, OnDeviceChangeCrash)):
(TestWebKitAPI::TEST(WebKit, EnumerateDevicesCrash)):
* Tools/TestWebKitAPI/Tests/WebKit/UserMessage.cpp:
(TestWebKitAPI::WebKit2UserMessageRoundTripTest::setInjectedBundleClient):
(TestWebKitAPI::WebKit2UserMessageRoundTripTest::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageConfiguration.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageCopySessionStateWithFiltering.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageGetScaleFactorNotZero.cpp:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WKPageIsPlayingAudio.cpp:
(TestWebKitAPI::setUpClients):
* Tools/TestWebKitAPI/Tests/WebKit/WKThumbnailView.mm:
(TestWebKitAPI::setPageLoaderClient):
* Tools/TestWebKitAPI/Tests/WebKit/WebArchive.cpp:
(TestWebKitAPI::setInjectedBundleClient):
(TestWebKitAPI::TEST(WebKit, WebArchive)):
* Tools/TestWebKitAPI/Tests/WebKit/WillLoad.cpp:
(TestWebKitAPI::WebKit2WillLoadTest::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/WillLoad_Bundle.cpp:
* Tools/TestWebKitAPI/Tests/WebKit/WillSendSubmitEvent.cpp:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/WillSendSubmitEvent_Bundle.cpp:
(TestWebKitAPI::WillSendSubmitEventTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/WebKit/mac/AttributedSubstringForProposedRangeWithImage.mm:
(TestWebKitAPI::TEST(WebKit, AttributedSubstringForProposedRangeWithImage)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/ContextMenuDownload.mm:
(TestWebKitAPI::TEST(WebKit, ContextMenuDownloadHTMLDownloadAttribute)):
(TestWebKitAPI::TEST(WebKit, ContextMenuDownloadHTMLDownloadAttributeWithSlashes)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/CustomBundleParameter.mm:
(TestWebKitAPI::TEST(WebKit, CustomBundleParameter)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/EditorCommands.mm:
(TestWebKitAPI::TEST(WebKit, ScrollByLineCommands)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/ForceLightAppearanceInBundle.mm:
(TestWebKitAPI::TEST(WebKit, ForceLightAppearanceInBundle)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/GetBackingScaleFactor.mm:
(TestWebKitAPI::setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKit/mac/GetPIDAfterAbortedProcessLaunch.cpp:
(TestWebKitAPI::TEST(WebKit, GetPIDAfterAbortedProcessLaunch)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/InjectedBundleAppleEvent.cpp:
(TestWebKitAPI::TEST(WebKit, InjectedBundleAppleEvent)):
* Tools/TestWebKitAPI/Tests/WebKit/mac/RestoreStateAfterTermination.mm:
(TestWebKitAPI::TEST(WebKit, RestoreStateAfterTermination)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/CommandBackForward.mm:
(WebKit2_CommandBackForwardTestWKView::SetUp):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/DecidePolicyForNavigationAction.mm:
(TEST(WebKit, DecidePolicyForNewWindowAction)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/PictureInPictureDelegate.mm:
(TestWebKitAPI::TEST(PictureInPicture, WKPageUIClient)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/ResponsivenessTimerDoesntFireEarly.mm:
(setInjectedBundleClient):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/UIDelegate.mm:
(TEST(WebKit, GeolocationPermission)):
((WebKit, GeolocationPermissionInIFrame)):
((WebKit, GeolocationPermissionInDisallowedIFrame)):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/WKURLSchemeHandler-1.mm:
((URLSchemeHandler, XHRPost)):
* Tools/TestWebKitAPI/Tests/WebKitGLib/TestUIClient.cpp:
* Tools/TestWebKitAPI/Tests/WebKitObjC/PreventImageLoadWithAutoResizing_Bundle.cpp:
(TestWebKitAPI::DenyWillSendRequestTest::didCreatePage):
* Tools/TestWebKitAPI/Tests/mac/FirstResponderScrollingPosition.mm:
(TestWebKitAPI::TEST(WebKit, DISABLED_FirstResponderScrollingPosition)):
* Tools/TestWebKitAPI/Tests/mac/FullscreenZoomInitialFrame.mm:
(TestWebKitAPI::FullscreenZoomInitialFrame::initializeView):
* Tools/TestWebKitAPI/Tests/mac/PageVisibilityStateWithWindowChanges.mm:
(TestWebKitAPI::PageVisibilityStateWithWindowChanges::initializeView):
* Tools/TestWebKitAPI/mac/WebKitAgnosticTest.mm:
(TestWebKitAPI::setPageLoaderClient):
* Tools/WebKitTestRunner/GeolocationProviderMock.cpp:
(WTR::GeolocationProviderMock::GeolocationProviderMock):

Canonical link: https://commits.webkit.org/287824@main
